src/integrations/admin/configuration-workout-integration.php

```diff
@@ -0,0 +1,192 @@
+<?php
+
+namespace Yoast\WP\SEO\Integrations\Admin;
+
+use WPSEO_Admin_Asset_Manager;
+use Yoast\WP\SEO\Conditionals\Admin_Conditional;
+use Yoast\WP\SEO\Helpers\Options_Helper;
+use Yoast\WP\SEO\Integrations\Integration_Interface;
+
+/**
+ * ConfigurationWorkoutsIntegration class
+ */
+class Configuration_Workout_Integration implements Integration_Interface {
+
+	/**
+	 * The admin asset manager.
+	 *
+	 * @var WPSEO_Admin_Asset_Manager
+	 */
+	private $admin_asset_manager;
+
+	/**
+	 * The options' helper.
+	 *
+	 * @var Options_Helper
+	 */
+	private $options_helper;
+
+	/**
+	 * {@inheritDoc}
+	 */
+	public static function get_conditionals() {
+		return [ Admin_Conditional::class ];
+	}
+
+	/**
+	 * Configuration_Workout_Integration constructor.
+	 *
+	 * @param WPSEO_Admin_Asset_Manager $admin_asset_manager The admin asset manager.
+	 * @param Options_Helper            $options_helper      The options helper.
+	 */
+	public function __construct(
+		WPSEO_Admin_Asset_Manager $admin_asset_manager,
+		Options_Helper $options_helper
+	) {
+		$this->admin_asset_manager = $admin_asset_manager;
+		$this->options_helper      = $options_helper;
+	}
+
+	/**
+	 * {@inheritDoc}
+	 */
+	public function register_hooks() {
+		\add_action( 'admin_enqueue_scripts', [ $this, 'enqueue_assets' ] );
+	}
+
+	/**
+	 * Adds the data for the configuration workout to the wpseoWorkoutsData object.
+	 */
+	public function enqueue_assets() {
+		// phpcs:ignore WordPress.Security.NonceVerification.Recommended -- Date is not processed or saved.
+		if ( ! isset( $_GET['page'] ) || $_GET['page'] !== 'wpseo_workouts' ) {
+			return;
+		}
+
+		$social_profiles = $this->get_social_profiles();
+
+		$this->admin_asset_manager->add_inline_script(
+			'workouts',
+			\sprintf(
+				'window.wpseoWorkoutsData["configuration"] = {
+				"isOrganization": %d,
+				"organizationName": "%s",
+				"organizationLogo": "%s",
+				"personId": %d,
+				"personLogo": "%s",
```

Comment:
I'd add `organizationLogoId` and `personLogoId` 
Also, it seems to me that internally we use `company` everywhere instead of Organization (even if this is what we display to the user) so maybe it's better to stick to `company` for consistency?